src/Simplex/Chat/Store/Messages.hs

```diff
@@ -2650,3 +2837,10 @@ getGroupHistoryItems db user@User {userId} GroupInfo {groupId} count = do
             LIMIT ?
           |]
           (userId, groupId, rcvMsgContentTag, sndMsgContentTag, count)
+
+divideFetchCountAround_ :: Int -> (Int, Int)
```

Comment:
this is overengineered and shouldn't be a function.
All it does is something like this:

before = count `div` 2
after = count - before

So it has to be called just in place

Reply:
to confuse things further, we pass count - 1 to this function